client/src/helpers/filters/filters.json

```diff
@@ -24,11 +24,11 @@
             "homepage": "https://github.com/AdguardTeam/AdGuardSDNSFilter",
             "source": "https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt"
         },
-        "adaway-default-blocklist": {
-            "name": "AdAway default blocklist",
+        "adaway": {
+            "name": "AdAway",
```

Comment:
I think the name was okay, AdAway is the name of the product, not blocklist

Reply:
Name updated